apps/src/sites/studio/pages/signup.js

```diff
@@ -127,6 +127,17 @@ window.SignupManager = function (options) {
     lastUserType = "teacher";
   }
 
+  function getSchoolDropdownStudyGroup() {
+    const variationID = 8256420202; // comes from Optimizely config
+    if (window.optimizely && window.optimizely.data && window.optimizely.data.state) {
```

Comment:
This might be an overly aggressive null check, I'm not sure if there are ever cases where optimizely will be there but data or data.state will not.